doc/release-notes-26899.md

```diff
@@ -0,0 +1,5 @@
+Updated settings
+----------------
+
+- Setting `-maxconnections=0` will now disable `-dnsseed`
+  and `-listen` (users may still set them to override).
```

Comment:
26899:
```suggestion
  and `-listen` (users may still set them to override) (#6649).
```
should maybe rename `doc/release-notes-26899.md` -> `doc/release-notes-6649.md`